Cordova.podspec

```diff
@@ -0,0 +1,32 @@
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
+  s.version      = "5.1.0"
+  s.summary      = "Cordova for iOS"
+  s.homepage     = "https://github.com/apache/cordova-ios"
+  s.license      = { :type => "Apache 2.0", :file => "LICENSE" }
+  s.author       = { "Bharath Hariharan" => "bhariharan@salesforce.com" }
+  s.platform     = :ios, "11.0"
```

Comment:
I think the minimum iOS version currently supported is 10.0, not 11.0

Reply:
Will do.